pkg/dflog/log.go

```diff
@@ -61,7 +61,7 @@ func WithLogFile(f string) Option {
 		if logger := getLumberjack(l); logger == nil {
 			l.SetOutput(&lumberjack.Logger{
 				Filename:   f,
-				MaxSize:    20, // mb
+				MaxSize:    40, // mb
```

Comment:
make it configurable? WDYT? And so do with the `MaxBackups`.

Reply:
agree. There's already a function named `WithMaxSizeMB` to configure `MaxSize`. It's better to open a new pr to implement this especially make the supernode's log configurable.